PointOfDispensingApp/src/brokerRequests.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+
+// define functions for API requests here
+export default {
+  // Search patient for patient retrieval
+  searchPatient: (data) => {
+    let query = `?family=${data.lastName}&given=${data.firstName}&birthdate=${data.birthDate}&address-postalcode=${data.postalCode}`;
+    return axios.get(`http://localhost:3000/Patient${query}`);
```

Comment:
Replace "http://localhost:3000" with "/broker" (for all instances).

Reply:
NOTE: you will need to add the following code the broker/server.js in the code wrapped by "if (process.env.DEVELOPMENT ...":

  // Reroute broker URLs from Vue app to actual endpoint (psuedo proxy).
  app.get('/broker/*', (req, res) => {
    req.url = req.url.replace(/^(\/broker)/, '');
    app.handle(req, res);
  });

Reply:
Done